tdvt/tdvt/version.py

```diff
@@ -1 +1 @@
-__version__ = '2.2.1'
+__version__ = '2.1.17'
```

Comment:
Make sure to pull `master` into your branch, we have some outdated code in it.

Reply:
I actually just pulled in the master branch to 2020.4 : https://github.com/tableau/connector-plugin-sdk/pull/659/commits/5391e46e10a64ac83dce211cca89b0a3254c9826  . Guess i chose the wrong line during merge conflict resolution. Will update the line